.murdock

```diff
@@ -339,7 +339,7 @@ static_tests() {
         exit 1
     fi
 
-    BUILDTEST_MCU_GROUP=static-tests ./dist/tools/ci/build_and_test.sh
+    # Travis will already run static tests
```

Comment:
Why not remove the whole function `static_tests()` altogether?

Reply:
BTW... I think this skips the rebase step included in the static tests.

Reply:
It seems like the function also takes care of cloning the repo.
TBH I'm not very familiar with the inner workings of Murdock, so my bet was to invoke Cunningham's Law on this one.

Reply:
> It seems like the function also takes care of cloning the repo.

Not entirely. The static tests script (```./dist/tools/ci/build_and_test.sh```) tests if the current branch can be rebased on master. It does so by rebaseing on upstream/master.
"static_tests()" just ensures that there's an upstream and it is correctly fetched.